### DISCUSSION TITLE
[Feature Request]: Delete Project/Site

### DISCUSSION THREAD

**Discussion by MimisK13** [beyondcode/herd-community#368] [Ideas]
### Feature Description

Add action to delete the project/site through Herd. 🙏

**Comment by nam-co** [comment]
I believe you can use herd unlink ... but it would be nice to have it in the UI

**Comment by isimmons** [comment]
First thing I looked for after creating a couple of sample sites. Now need to get rid of them and start working for real. 

So I believe an action to delete and/or a detailed document for manual deletion would be great in the future. Here is my experience figuring it out.

Not sure what the "name" is supposed to be for the unlink command so I ran `herd parked` and then used the "Site" name.

Not sure where these "links" are stored.

```
C:\Users\isimm>herd parked
+-------------+-----+-------------------------+---------------------------------+-----+
| Site        | SSL | URL                     | Path                            |     |
+-------------+-----+-------------------------+---------------------------------+-----+
| example-app |     | http://example-app.test | C:\Users\isimm\Herd\example-app | 8.3 |
| foobar      |     | http://foobar.test      | C:\Users\isimm\Herd\foobar      | 8.3 |
+-------------+-----+-------------------------+---------------------------------+-----+

C:\Users\isimm>herd unlink example-app
The [example-app] symbolic link has been removed.

C:\Users\isimm>herd unlink foobar
The [foobar] symbolic link has been removed.
```

But the sites still gets served and works in the browser. I know very little about valet and guess I need to learn.

So then I deleted the actual projects from the Herd directory and I see that Herd did automatically remove them from my hosts file when the project folders get deleted.

So I assume at this point that every trace of the site/project is gone. But is it? Did I do the right thing with unlink? Is there anything else floating around?
